src/core/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -753,4 +753,10 @@
   <data name="GeneralSettings_VersionIsLatest" xml:space="preserve">
     <value>You have the latest available version.</value>
   </data>
+  <data name="FileExplorerPreview_IconThumbnail_GroupSettings.Text" xml:space="preserve">
+    <value>Icon Preview</value>

```

Comment:
[nit] since the text below is SVG thumbnails, is Thumbnail Preview more appropriate? I'm not too familiar with this space.

Reply:
depends how purist someone wants to be.  What we view as technically correct causes confusion for others.  if people file  a bug against it, lets revisit